cgmanifest.json

```diff
@@ -0,0 +1,57 @@
+{
+    "Registrations": [ 
+        {
+            "Component": { 
+                "Type": "git", 
+                "git": {
+                    "RepositoryUrl": "https://github.com/rxi/log.c",
+                    "CommitHash": "f9ea34994bd58ed342d2245cd4110bb5c6790153"
+                }
+            },
+            "DevelopmentDependency": false
+        },
+        {
+            "Component": { 
+                "Type": "git", 
+                "git": {
+                    "RepositoryUrl": "https://github.com/kgabis/parson",
+                    "CommitHash": "8beeb5ea4da5eedff8d3221307ef04855804a920"
+                }
+            },
+            "DevelopmentDependency": false
+        },
+        {
+            "Component": { 
+                "Type": "git", 
+                "git": {
+                    "RepositoryUrl": "https://github.com/postgres/postgres",
+                    "CommitHash": "9213462c539e6412fe0498a7f8e20b662e15c4ec"
+                }
+            },
+            
+            "license": "PostgreSQL",
+            "licenseDetail": [
```

Comment:
For some reason, NOTICE generation doesn't automatically add this definition. But we can manually append this to the NOTICE file. It is probably okay for it to be not automated as we will generate the notice file only once.